tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java

```diff
@@ -46,4 +51,24 @@ private Long parseRunId(String id) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    @Override
+    public TesServiceInfo getServiceInfo() {
+        String nameOfService = "CloudPipeline";
+        String doc = "https://epam.github.io/cloud-pipeline/";
+        TesServiceInfo tesServiceInfo = new TesServiceInfo();
+        tesServiceInfo.setName(nameOfService);
+        tesServiceInfo.setDoc(doc);
+        tesServiceInfo.setStorage(getDataStorage());
+        return tesServiceInfo;
+    }
+
+    private List<String> getDataStorage(){
+        List<String> listPathDataStorage = new ArrayList<>();
```

Comment:
Let's use Stream API here:
`cloudPipelineAPIClient.loadAllDataStorages().stream().map( ... ).collect( ... )`
Also check that `cloudPipelineAPIClient.loadAllDataStorages()` doesn't return `null`. Convenient way to use it `ListUtils.emptyIfNull` method